dll/win32/comctl32/edit.c

```diff
@@ -3469,6 +3469,9 @@ static LRESULT EDIT_WM_KeyDown(EDITSTATE *es, INT key)
                 {
                     if (!notify_parent(es, EN_UPDATE)) break;
                     notify_parent(es, EN_CHANGE);
+#ifdef __REACTOS__
```

Comment:
wait, this is a winesync yet you add an `#ifdef __REACTOS__` ?

Reply:
I didn't add at first but @SergeGautherie, @whindsaks and @HBelusca asked me to. I don't know the exact reason but seems like without this it would make problems for WINE when they're cherry-picking our changes.

Reply:
Not HBelusca but others did think this way it's better.
Link to the discussion: https://github.com/reactos/reactos/pull/7940#discussion_r2061640790